test/e2e/test/apmserver/builder.go

```diff
@@ -183,10 +190,26 @@ func (b Builder) WithoutIntegrationCheck() Builder {
 	}
 
 	return b.WithConfig(map[string]interface{}{
-		"apm-server.data_streams.wait_for_integration": false,
+		APMServerDataStreamsWaitForIntegration: false,
 	})
 }
 
+func (b Builder) HasNoIntegration() bool {
```

Comment:
This feels as though it would be easier to read if it was `HasIntegration`, but since it's simply the e2e tests, 👍 .